src/net.cpp

```diff
@@ -1670,12 +1671,12 @@ void CConnman::ThreadOpenConnections(const std::vector<std::string> connect)
                 LogPrintf("Adding fixed seeds as 60 seconds have passed and addrman is empty for at least one reachable network\n");
             }
 
-            // Checking !dnsseed is cheaper before locking 2 mutexes.
+            // Checking !dnsseed is cheaper before locking a mutex.
```

Comment:
Checking `use_seednodes` is cheap too, so should also be checked before acquiring the mutex rather than after.

Reply:
fixed